lib/matplotlib/tests/test_datetime.py

```diff
@@ -318,11 +318,40 @@ def test_spy(self):
         fig, ax = plt.subplots()
         ax.spy(...)
 
-    @pytest.mark.xfail(reason="Test for stackplot not written yet")
     @mpl.style.context("default")
     def test_stackplot(self):
-        fig, ax = plt.subplots()
-        ax.stackplot(...)
+        mpl.rcParams["date.converter"] = 'concise'
+        N = 10
+        fig, (ax1, ax2) = plt.subplots(2, 1, constrained_layout=True)
+
+        stacked_nums = np.array([range(1, N)] * 4)
+
+        dates_days = np.array(
+            [datetime.datetime(2023, 1, i) for i in range(1, N)]
+        )
+        dates_years = np.array(
+            [datetime.datetime(1970 + i, 1, 1) for i in range(1, N)]
```

Comment:
I would avoid this date, as 1970 can hide problems where we don't actually convert dates properly.

(You are _actually_ avoiding the most likely problematic year by starting at 1, but it is still close enough to throw some alarm bells off)